examples/express-ts/package.json

```diff
@@ -21,7 +21,7 @@
     "knex": "^0.14.2",
     "lodash": "^4.17.4",
     "morgan": "^1.9.0",
-    "objection": "^0.9.2",
+    "objection": "../..",
```

Comment:
I needed to restore the `../..` to pull in the typings changes in this same PR. I believe clean rebuilds of node_modules would have fixed the prior issue, as my scratch build is fine.